xanalytics/management/commands/populate_xanalytics.py

```diff
@@ -0,0 +1,42 @@
+"""Management command to populate xanalytics data by course ID."""
+
+import logging
+from random import randint
```

Comment:
It's possible. However, since courses can't be imported twice it wouldn't make a difference in the CI environment, and in tests all that matters is that they now have non-zero values. In short, I don't see any benefit, and it's slightly more code.